backend/controller/tasks.py

```diff
@@ -3,5 +3,5 @@
 
 
 @celery.task(track_started=True)
-def optimize_model(url):
-    return Optimize().run(url)
+def optimize_model(url, model_type):
+    return Optimize({'cpu': 2, 'gpu':0}, model_type).run(url)
```

Comment:
Esto aqui no esta bien, en este caso el default ya es eso. Necesitamos solo pasarle los cpu/gpu dependiendo lo del server so it would be .env variables here and not static variables.

Reply:
return Optimize(resources=env.resources, model_type=model_type).run(url) should be the correct approach here

Reply:
done